.travis.yml

```diff
@@ -1,14 +1,16 @@
+dist: trusty
 sudo: false
 language: node_js
 cache:
   directories:
     - ~/.npm
 notifications:
   email: false
-node_js: '8'
-install: npm install
-script: npm test
+before_install:
+  - npm i -g npm@latest
 after_success:
-  - npx -p semantic-release@7 -c "semantic-release pre && npm publish && semantic-release post"
+  - npm run travis-deploy-once "npm run semantic-release"
```

Comment:
`npm run` means a `runScript`, but I don't see it in `scripts`? This looks odd.

Reply:
forgot to put them in...